src/components/home.jsx

```diff
@@ -12,7 +12,7 @@ const Home = () => {
       setIsLoading(false);
     });
   }, []);
-  if (isLoading) return <p>Loading...</p>;
+  if (isLoading) return <p>🐢 Loading...🐢</p>;
```

Comment:
love the emojis!

whereever you use them be sure to consider accessibility

best practice is to wrap them in a `<span>` with some appropriate tags

> if you want them to be announced by screen readers
```jsx
  if (isLoading) return <p><span role="img" aria-label="loading-turtle">🐢</span> Loading...<span role="img" aria-label="loading-turtle">🐢</span></p>;
```

> if you want them to be ignored (just read out "loading...")
```jsx
  if (isLoading) return <p><span role="img" aria-hidden={true}>🐢</span> Loading...<span role="img" aria-hidden={true}>🐢</span></p>;
```